src/server/enrollment-system.js

```diff
@@ -63,35 +63,97 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
+function zeroPadNumber(number, padding) {
```

Comment:
Can you add a comment explaining what this does?  Use JSDoc format (@param and @return) to give an example of input and output.